docs/api-reference/carto/quadbin-tile-layer.md

```diff
@@ -73,7 +73,7 @@ Use one of the following [Data Sources](./data-sources.md) to fetch this from th
 
 ### Data Accessors
 
-##### `getQuadbin` ([Function](../../developer-guide/using-layers.md#accessors), optional) {#getquadkey}
+##### `getQuadbin` ([Function](../../developer-guide/using-layers.md#accessors), optional) {#getquadbin}
```

Comment:
Appears to be a dead link immediately below this as well - perhaps we remove that link?

Reply:
Not sure what you mean, https://deck.gl/docs/api-reference/geo-layers/quadkey-layer#getquadkey is valid

Reply:
Ah, it's broken in GitHub's markdown preview mode – I guess I needed to run the website locally to check these links sorry!